app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -169,6 +165,9 @@ public void onUpgrade(SQLiteDatabase database, ConnectionSource connectionSource
                 Log.e(TAG, "Upgrading failed, the column firstLearnData might already exists.", e);
             }
         }
+        if (oldVersion <= 5) {
```

Comment:
How did you change the version?

Reply:
there is a variable keeping track of the version, it was set to 5, now it is set to 6.